File: node_10.18.1/upload_items.js
(async () => {
	
	var 
		FS_PROMISE = require("fs").promises, //as we have v10
		AWS_SDK = require("aws-sdk"),
		S3_API = new AWS_SDK.S3({
			apiVersion: "2006-03-01",
			region: "us-west-2"
		}),
		BUCKET_NAME_STR = "<FMI>", //SWAP THIS OUT FOE YOUR BUCKET NAME
		//e.g rh-2020-02-25-s3site
		params = {},
		promise_arr = [],
		FILES_ARR = [ // lexical order of all know website files
			"backdrop_camera.jpg",
			"config.js",
			"core.css",
			"flex_search.js",
			"index.html",
			"jquery.js",
			"kiosk.png",
			"main.css",
			"main.js",
			"products.js",
			"report.html",
			"reset.css",
			"search.css",
			"search.js"
		],
		CONTENT_TYPE_PARALLEL_ARR = [ // not self discovering for some reason, hence needed
			"image/jpg",
			"application/javascript",
			"text/css",
			"application/javascript",
			"text/html",
			"application/javascript",
			"image/png",
			"text/css",
			"application/javascript",
			"application/javascript",
			"text/html",
			"text/css",
			"text/css",
			"application/javascript"
		],
		WEBSITE_FOLDER_PATH_STR = "/home/ec2-user/environment/resources/website/";
	
	params.Bucket = BUCKET_NAME_STR;

	for(var i_int = 0; i_int < FILES_ARR.length; i_int += 1){
		params.Key = FILES_ARR[i_int];
		params.ContentType = CONTENT_TYPE_PARALLEL_ARR[i_int];
		params.CacheControl = "max-age=0";
		params.Body = await FS_PROMISE.readFile(WEBSITE_FOLDER_PATH_STR + FILES_ARR[i_int]); 
		promise_arr.push(S3_API.putObject(params).promise());
	}

	try{
		await Promise.all(promise_arr);
		//returns and array of 7 Etags
		console.log("Done");
	}catch(e){
			console.error("Problem", e);
	}

})();

File: python_3.6.8/upload_items.py
'''
	You must replace <FMI> with your bucket name
'''
import boto3
S3API = boto3.client("s3", region_name="us-west-2") 
bucket_name = "ar-2021-01-23-s3site"

filename = "/home/ec2-user/environment/resources/website/backdrop_camera.jpg"
S3API.upload_file(filename, bucket_name, "backdrop_camera.jpg", ExtraArgs={'ContentType': "image/jpg", "CacheControl": "max-age=0"})

filename = "/home/ec2-user/environment/resources/website/config.js"
S3API.upload_file(filename, bucket_name, "config.js", ExtraArgs={'ContentType': "application/js", "CacheControl": "max-age=0"})

filename = "/home/ec2-user/environment/resources/website/core.css"
S3API.upload_file(filename, bucket_name, "core.css", ExtraArgs={'ContentType': "text/css", "CacheControl": "max-age=0"})

filename = "/home/ec2-user/environment/resources/website/flex_search.js"
S3API.upload_file(filename, bucket_name, "flex_search.js", ExtraArgs={'ContentType': "application/js", "CacheControl": "max-age=0"})

filename = "/home/ec2-user/environment/resources/website/index.html"
S3API.upload_file(filename, bucket_name, "index.html", ExtraArgs={'ContentType': "text/html", "CacheControl": "max-age=0"})

filename = "/home/ec2-user/environment/resources/website/jquery.js"
S3API.upload_file(filename, bucket_name, "jquery.js", ExtraArgs={'ContentType': "application/js", "CacheControl": "max-age=0"})

filename = "/home/ec2-user/environment/resources/website/kiosk.png"
S3API.upload_file(filename, bucket_name, "kiosk.png", ExtraArgs={'ContentType': "image/png", "CacheControl": "max-age=0"})

filename = "/home/ec2-user/environment/resources/website/main.css"
S3API.upload_file(filename, bucket_name, "main.css", ExtraArgs={'ContentType': "text/css", "CacheControl": "max-age=0"})

filename = "/home/ec2-user/environment/resources/website/main.js"
S3API.upload_file(filename, bucket_name, "main.js", ExtraArgs={'ContentType': "application/js", "CacheControl": "max-age=0"})

filename = "/home/ec2-user/environment/resources/website/products.js"
S3API.upload_file(filename, bucket_name, "products.js", ExtraArgs={'ContentType': "application/js", "CacheControl": "max-age=0"})

filename = "/home/ec2-user/environment/resources/website/reset.css"
S3API.upload_file(filename, bucket_name, "reset.css", ExtraArgs={'ContentType': "text/css", "CacheControl": "max-age=0"})

filename = "/home/ec2-user/environment/resources/website/search.css"
S3API.upload_file(filename, bucket_name, "search.css", ExtraArgs={'ContentType': "text/css", "CacheControl": "max-age=0"})


print ("DONE")